index.php

```diff
@@ -353,7 +353,7 @@
 
 	// "rawContent" is especially meant for technical endpoints.
 	// This endpoint doesn't need any theme initialization or other comparable stuff.
-	if (!$a->error && $a->module_class) {
+	if (!$a->error && $a->module_class && method_exists($a->module_class, 'rawContent')) {
```

Comment:
This is better solved by defining the `rawContent` method in the `BaseModule` class.